boards/FSM/Inc/App/configs/App_FlowRateThresholds.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#define PRIMARY_FLOW_METER_MIN_FLOW_RATE 1.0f
```

Comment:
A comment here linking how the datasheets indicate the range is 1-30 L/min would be nice

Reply:
done